### PR TITLE
Feat/throwables

### PR DESCRIPTION
**Pull request by Darthmaul**
Implements #339 

This functionality is to implement a new component to the existing Builder & Preview pages. Additionally, I have added a new search option on the browse page for our users to find loadouts with specific throwables.

I will just need to update the icon database value on Prod once we're ready to push. A staging review is welcomed.

Screenshots attached, feedback welcomed.

image (4)
image (3)
image (5)
image
image

### CODE REVIEW

File: resources/scss/_builder.scss [hunk 1]
```diff
@@ -227,6 +227,9 @@
     .hundredPercent {
         width: 100%;
     }
+    .fiftyPercent {
+        width: 50%;
+    }
```
Comment (chaseconey) [inline resources/scss/_builder.scss:232]: Tailwind has a native class for this, `w-1/2`.

https://tailwindcss.com/docs/width

**Review by chaseconey — CHANGES_REQUESTED** [review]
Couple minor things.

The other thing that we could address is making the `throwable_id` nullable. That would fix the issue that you are describing in Slack as far as I am following.

I am not sure any of this logic would even have to change, the gql endpoint would just give a null value for that field. Thoughts?

File: resources/js/apolloQueries.js [hunk 2]
```diff
@@ -174,6 +174,20 @@ export default {
...
+                          description
+                          icon
+                          json_stats
+                            credits_cost
+                            magnite_cost
+                            bismor_cost
+                            umanite_cost
+                            croppa_cost
+                            enor_pearl_cost
+                            jadiz_cost
```
[the remark this answers is not in the document]
Comment (Darthmaul) [reply resources/js/apolloQueries.js]: Fixed - thanks

File: resources/scss/_builder.scss (hunk 1 above)
Comment (Darthmaul) [reply resources/scss/_builder.scss:232]: Fixed - good catch, thanks

**Comment by Darthmaul** [comment]
Screenshot of new builder view thanks to the stats we will have - stats already in staging
image

**Review by chaseconey — APPROVED** [review]

**Pull request merged**